toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -587,27 +574,25 @@ func (r *RpmRepoCloner) clonePackage(baseArgs []string, enabledRepoOrder ...stri
 }
 
 func convertPackageVersionToTdnfArg(pkgVer *pkgjson.PackageVer) (tdnfArg string) {
-	tdnfArg = pkgVer.Name
 	// TDNF does not accept versioning information on implicit provides.
 	if pkgVer.IsImplicitPackage() {
 		if pkgVer.Condition != "" {
 			logger.Log.Warnf("Discarding version constraint for implicit package: %v", pkgVer)
 		}
+		tdnfArg = pkgVer.Name
 		return
 	}
 
 	// Treat <= as =
```

Comment:
Comment is no longer relevant

Reply:
If you mean the implicit provides comment, tdnf still completely ignores it unfortunately.